processes/migration.py

```diff
@@ -1,5 +1,6 @@
-from alembic import config
+import newrelic.agent
```

Comment:
Would also skip this process! This is maintenance which also generally doesn't tell us much about the app. Anytime this gets run I expect you/I/another dev to be watching it and be able to report on what happens.